boot.php

```diff
@@ -647,7 +647,7 @@ function killme()
 /**
  * @brief Redirect to another URL and terminate this process.
  */
-function goaway($path)
+function goaway($path = '')
```

Comment:
Interesting for #5907 
So I will rename it from `System::redirectTo($url)` to `System::redirect($to = '');` :-)